human-code.md

```diff
@@ -2,9 +2,9 @@ Human Code
 ==========
 What's human code?
 ------------------
-Imagine you're an airline and you have a sequence of things that need to be done to make a plane ready before takeoff. Would you rely on the pilot to remember these things? Of course not - the human memory is prone to lapses and affected by even things as small as what you ate that day; missing even one thing could spell disaster in flight. Even if one pilot remembered the steps perfectly, the next pilot might not. No, your airline would make a checklist.
+Imagine you're an airline and you have a sequence of things that need to be done to make a plane ready before takeoff. Would you rely on the pilot to remember these things? Of course not - the human memory is prone to lapses and distracted by things as small as what you ate that day; missing even one thing could spell disaster in flight. Where one pilot remembered the steps perfectly, the next pilot might not. No, your airline would make a checklist.
```

Comment:
Mostly reducing the repetition of "even"

Reply:
Good catch!!